PyInstaller/utils/osx.py

```diff
@@ -101,13 +101,35 @@ def get_macos_sdk_version(filename):
     header = binary.headers[0]
     # Find version command using helper
     version_cmd = _find_version_cmd(header)
+    return _hex_triplet(version_cmd[1].sdk)
+
+
+def _hex_triplet(version):
     # Parse SDK version number
-    major = (version_cmd[1].sdk & 0xFF0000) >> 16
-    minor = (version_cmd[1].sdk & 0xFF00) >> 8
-    revision = (version_cmd[1].sdk & 0xFF)
+    major = (version & 0xFF0000) >> 16
+    minor = (version & 0xFF00) >> 8
+    revision = (version & 0xFF)
     return major, minor, revision
 
 
+def macosx_version_min(filename: str) -> tuple:
```

Comment:
As a minor nitpick, I would name this `get_macosx_version_min`.

Reply:
Ah, you'll have to fight me on that one because we're at odds! I personally hate calling things `get_xxx()`, `set_xxx()`, `is_xxx()` or `has_xxx()`.

Reply:
Fair enough.